### DISCUSSION TITLE
Welcome to Ultra-Fast-Lane-Detection Discussions!

### DISCUSSION THREAD

**Discussion by cfzd** [cfzd/Ultra-Fast-Lane-Detection#143] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ayushupneja** [comment]
How did you manage to get the demo working? How can I input an image/video and receive the lane output? I was able to get the model working on the TuSimple dataset but was interested in implementing it with a live camera feed. @cfzd @zchrissirhcz

**Comment by wwandji** [comment]
i would like to build my own dataset like tusimple dataset, but i have a question, how they do to generate the json file(label_data_0313.json, label_data_0531.json, label_data_0601.json , test_tasks_0627.json and test_label.json)

i will be happy for a answer @cfzd @zchrissirhcz @liej6799 @ayushupneja
